### PR TITLE
Remove admin

### PR DESCRIPTION
**Pull request by azharmalik3**
@mosic

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 require File.expand_path('../boot', __FILE__)
 
 require "action_controller/railtie"
-require "active_record/railtie"
```
Comment (mosic) [inline config/application.rb]: This causes all tests to fail. Either revert it or add it to the test runner.

File: Gemfile [hunk 2]
```diff
@@ -18,7 +18,6 @@ gem 'stripe',
   github: 'stripe/stripe-ruby'
 gem 'stripe_event'
 
-gem 'pg'
```
Comment (mosic) [inline Gemfile]: We're still using the database for OAuth, so I don't there's a reason to remove the `pg` gem.

**Comment by azharmalik3** [comment]
@mosic Added `pg` gem and `ActiveRecord` again in project. Also fixed broken tests.

**Comment by mosic** [comment]
@azharmalik3 Thanks, nice work :+1:

**Pull request merged**